src/Raven.Server/ServerWide/ClusterStateMachine.cs

```diff
@@ -49,6 +50,15 @@ public class ClusterStateMachine : RachisStateMachine
     {
         private const string LocalNodeStateTreeName = "LocalNodeState";
         private static readonly TableSchema ItemsSchema;
+
+        private static readonly TableSchema UniqueItemsSchema;
```

Comment:
I think we scan the entire voron file and send the structures with the relevant name. 
Which in this case I've named them both 'Items'. ( Not sure though if I need different names)